pagecarton/core/library/Ayoola/Application.php

```diff
@@ -1353,27 +1336,27 @@ function_exists( 'http_response_code' ) ? http_response_code(404) : null;
 			//	var_export( $PAGE_INCLUDE_FILE );
 			//	var_export( $pageThemeFileUrl );
 			//	exit();
-		
+
 				// intended copy next
 				$intendedCopyPaths = Ayoola_Page::getPagePaths( '/' . trim( $uri . '/default', '/' ) );
 				$PAGE_INCLUDE_FILE = Ayoola_Loader::getFullPath( $intendedCopyPaths['include'], array( 'prioritize_my_copy' => true ) );
 				$PAGE_TEMPLATE_FILE = Ayoola_Loader::getFullPath( $intendedCopyPaths['template'], array( 'prioritize_my_copy' => true ) );
 			//	var_export( $intendedCopyPaths['include'] );
 			//	var_export( $intendedCopyPaths['template'] );
 				if
-				( 
-					! $PAGE_INCLUDE_FILE OR ! $PAGE_TEMPLATE_FILE
+				(
+					! $PAGE_INCLUDE_FILE AND ! $PAGE_TEMPLATE_FILE
 				)
 				{
-					
+
 					//	global copy
 					$PAGE_INCLUDE_FILE = Ayoola_Loader::getFullPath( $pagePaths['include'], array( 'prioritize_my_copy' => true ) );
 					$PAGE_TEMPLATE_FILE = Ayoola_Loader::getFullPath( $pagePaths['template'], array( 'prioritize_my_copy' => true ) );
 				//	var_export( $pagePaths['include'] );
 				//	var_export( $pagePaths['template'] );
 					if
-					( 
-						! $PAGE_INCLUDE_FILE OR ! $PAGE_TEMPLATE_FILE
+					(
+						! $PAGE_INCLUDE_FILE AND ! $PAGE_TEMPLATE_FILE

```

Comment:
Both files must be available for the page to work. So I think the logic should be `OR`